docs/source/optimization_inc.mdx

```diff
@@ -126,6 +126,33 @@ mpirun -np <number_of_processes> <RUN_CMD>
 
 Please refer to INC [documentation](https://github.com/intel/neural-compressor/blob/master/docs/source/tuning_strategies.md#distributed-tuning) and [text-classification](https://github.com/huggingface/optimum-intel/tree/main/examples/neural_compressor/text-classification) example for more details. 
 
+## Weight-only quantization
```

Comment:
Let's wait for this feature to be more stable before adding it to the documentation (currently not compatible with optimum-intel latest release)